datafusion/sql/src/expr/order_by.rs

```diff
@@ -30,6 +30,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         exprs: &[OrderByExpr],
         schema: &DFSchema,
         planner_context: &mut PlannerContext,
+        literal_to_column: bool,
```

Comment:
I think this is subtle enough that it would be good if we can add a comment explaining what the parameter does

Perhaps something like 

```rust
    /// convert sql [OrderByExpr] to `Vec<Expr>`
    ///
    /// if `literal_to_column` is true, treat any numeric literals (e.g.  `2`) as a 1 based index
    /// into the SELECT list (e.g. `SELECT a, b FROM table ORDER BY 2`).
    /// If false, interpret numeric literals as constant values
    pub(crate) fn order_by_to_sort_expr(
```

Reply:
Yea, added code comment as suggested.